gui/widgets/matRad_DVHWidget.m

```diff
@@ -1,51 +1,61 @@
 classdef matRad_DVHWidget < matRad_Widget
```

Comment:
There's one issue I have with the DVH-widget and the stats widget in its current implementation - you should somehow be able to also put in already existing dvh objects.

I would also precache already computed DVH objects in some property, maybe this can be combined.